contracts/src/v0.8/ccip/test/feeQuoter/FeeQuoter.getTokenPrice.t.sol

```diff
@@ -8,7 +8,7 @@ import {FeeQuoterSetup} from "./FeeQuoterSetup.t.sol";
 
 contract FeeQuoter_getTokenPrice is FeeQuoterSetup {
   function test_GetTokenPriceFromFeed() public {
-    uint256 originalTimestampValue = block.timestamp;
+    uint32 originalTimestampValue = uint32(block.timestamp);
```

Comment:
This is needed because via-ir optimizes block.timestamp too much, meaning warp doesn't work well unless you cast the previous value of block.timestamp making it a value ref instead of pointer ref (to use offchain terms)

Reply:
q: are we compiling test files with via-ir for geth wrappers generation?

Reply:
No, but we want to be able to run the testsuite with via-ir before releases, so the tests should still be compatible with via-ir